packages/manager/apps/pci-rancher/src/api/api.type.ts

```diff
@@ -27,6 +27,7 @@ export interface PciProject {
   project_id: string;
   status: string;
   unleash: boolean;
+  planCode: 'project.discovery' | 'project.2018';
```

Comment:
**nitpick:** This can be hard to maintain when new Plan-code is added on PU side. 
```suggestion
  planCode: string;
```